src/Components/App/App.js

```diff
@@ -72,4 +79,9 @@ function App() {
   );
 }
 
+App.propTypes = {
+  error: PropTypes.string,
+  cocktail: PropTypes.array,
```

Comment:
I think you could even get a little more specific if you wanted by using this `// An array of a certain type
  optionalArrayOf: PropTypes.arrayOf(PropTypes.number)`